Python/Product/PythonTools/PythonTools/Options/PythonInterpreterOptionsControl.cs

```diff
@@ -60,6 +60,10 @@ public PythonInterpreterOptionsControl()
                 _defaultInterpreter.Items.Clear();
 
                 foreach (var interpreter in _serviceProvider.GetPythonToolsService().InterpreterOptions.Select(x => x.Key).OrderBy(f => f.Description)) {
+                    if (!Microsoft.PythonTools.Interpreter.PythonInterpreterFactoryExtensions.IsUIVisible(interpreter)) {

```

Comment:
There's a naming collision that prevents the extension method from being resolved properly. Looks like I can rename one of the static classes to resolve it though...